packages/contracts-bedrock/deploy-config/goerli.ts

```diff
@@ -1,15 +1,15 @@
 import { ethers } from 'ethers'
 
 const sequencerAddress = '0x6c23a0dcdfc44b7a57bed148de598895e398d984'
-const startingTimestamp = 1658777882
+const l1StartingBlockTag = ''
```

Comment:
Why is this empty? Should we find the tag that corresponds with the previous `startingTimestamp`?

Reply:
`cast find-block <timestamp>`

Reply:
fixed (had to use block hash from k8s repo)